Sources/SpeziOnboarding/SignatureView.swift

```diff
@@ -109,45 +114,119 @@ public struct SignatureView: View {
     ///   - isSigning: A `Binding` indicating if the user is currently signing.
     ///   - canvasSize: The size of the canvas as a Binding.
     ///   - name: The name that is displayed under the signature line.
+    ///   - formattedDate: The formatted date that is displayed under the signature line.
     ///   - lineOffset: Defines the distance of the signature line from the bottom of the view. The default value is 30.
-    init(
+    public init(
         signature: Binding<PKDrawing> = .constant(PKDrawing()),
         isSigning: Binding<Bool> = .constant(false),
         canvasSize: Binding<CGSize> = .constant(.zero),
         name: PersonNameComponents = PersonNameComponents(),
+        formattedDate: String? = nil,
         lineOffset: CGFloat = 30
     ) {
         self._signature = signature
         self._isSigning = isSigning
         self._canvasSize = canvasSize
         self.name = name
+        self.formattedDate = formattedDate
         self.lineOffset = lineOffset
     }
+
+    /// Creates a new instance of an ``SignatureView``  with `String`-based name components.
+    /// - Parameters:
+    ///   - signature: A `Binding` containing the current signature as an `PKDrawing`.
+    ///   - isSigning: A `Binding` indicating if the user is currently signing.
+    ///   - canvasSize: The size of the canvas as a Binding.
+    ///   - givenName: The given name that is displayed under the signature line.
+    ///   - familyName: The family name that is displayed under the signature line.
+    ///   - formattedDate: The formatted date that is displayed under the signature line.
+    ///   - lineOffset: Defines the distance of the signature line from the bottom of the view. The default value is 30.
+    public init(
+        signature: Binding<PKDrawing> = .constant(PKDrawing()),
+        isSigning: Binding<Bool> = .constant(false),
+        canvasSize: Binding<CGSize> = .constant(.zero),
```

Comment:
Wondering if we should move the communication of all that information as well as the communication of the consent document PDF from the views where we currently use Bindings to preferences? Not something to be done in this PR, but I would like to explore what other SwiftUI/Apple views out there doing similar complex work are using to communicate all this information upstream. I think the setup we have is fine, but even just the discussion above with the enum is prompting some thoughts if all of the needs to be embedded in larger binding-like structures.

Reply:
Thank you!

Reply:
See #64